tests/testthat/model-examples/babylon.yaml

```diff
@@ -0,0 +1,29 @@
+babylon_binary: /data/apps2/bbi
```

Comment:
TL;DR `git rm` this file and maybe even put a `.gitignore` in this folder with `babylon.yaml` in it.

Longer:
My gut is that this file should _not_ be checked in. The only reason we would need it is for re-running the model again and I think that, in that case, we should have to do `bbi init` (to create this file) again too. 

Mainly because the main reason we will re-run the model is when `bbi` has changed and in that case we would want an updated "default" `babylon.yaml` from the new `bbi` too.

Reply:
File removed. We can discuss how to handle re-running the test model (including what to do with `babylon.yaml`) in #142.